man/systemd-stub.xml

```diff
@@ -188,9 +189,9 @@
       measured into TPM PCR 12 (if a TPM is present).</para></listitem>
 
       <listitem><para>Additionally, files <filename>/loader/addons/*.addon.efi</filename> are loaded and
-      verified as PE binaries, and a <literal>.cmdline</literal> section is parsed from them. This is
-      supposed to be used to pass additional command line parameters to the kernel, regardless of the kernel
-      being booted.</para></listitem>
+      verified as PE binaries, and <literal>.cmdline</literal> and/or <literal>.dtb</literal> sections are
+      parsed from them. This is supposed to be used to pass additional command line parameters or device
+      tree blobs to the kernel, regardless of the kernel being booted.</para></listitem>
```

Comment:
the dt spec spells this as "Devicetree" btw, not "device tree" or "Device Tree" or "DeviceTree" or so.